js/index.js

```diff
@@ -1,42 +1,86 @@
+let cartProduct = document.querySelector('tbody');
+
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  const price = +product.querySelector('.price span').innerText;
+  const quantity = +product.querySelector('.quantity input').value;
+  let subtotal = product.querySelector('.subtotal span');
+  subtotal.innerText = price * quantity;
+  return subtotal.innerText;
 }
 
 function calculateAll() {
   // code in the following two lines is added just for testing purposes.
   // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
+  // const singleProduct = document.querySelector('.product');
+  // updateSubtotal(singleProduct);
   // end of test
 
   // ITERATION 2
   //... your code goes here
-
+  const row1 = document.querySelector('.product');
+  updateSubtotal(row1);
+  const row2 = document.querySelector('.product2');
+  updateSubtotal(row2);
   // ITERATION 3
-  //... your code goes here
+  //... your code goes here total-value
+  let total = document.querySelector("[id='total-value'] span");
```

Comment:
```suggestion
  let total = document.querySelector("#total-value span");
```
Esto seria lo mismo!, pero lo que habias puesto estaba bien :D